make_check_initializer/griddb_init.c

```diff
@@ -0,0 +1,542 @@
+                  3,
+                  "a", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NOT_NULL,
+                  "b", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NULLABLE,
+                  "c", GS_TYPE_STRING,      GS_TYPE_OPTION_NULLABLE);
+  if (ret != GS_RESULT_OK)
+    return ret;
+  
+  ret = set_tableInfo(store, "fprt1_p2", &fprt1_p2,
+                  3,
+                  "a", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NOT_NULL,
+                  "b", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NULLABLE,
+                  "c", GS_TYPE_STRING,      GS_TYPE_OPTION_NULLABLE);
+  if (ret != GS_RESULT_OK)
+    return ret;
+  
+  ret = set_tableInfo(store, "fprt2_p1", &fprt2_p1,
+                  3,
+                  "a", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NOT_NULL,
+                  "b", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NULLABLE,
+                  "c", GS_TYPE_STRING,      GS_TYPE_OPTION_NULLABLE);
+  if (ret != GS_RESULT_OK)
+    return ret;
+  
+  ret = set_tableInfo(store, "fprt2_p2", &fprt2_p2,
+                  3,
+                  "a", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NOT_NULL,
+                  "b", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NULLABLE,
+                  "c", GS_TYPE_STRING,      GS_TYPE_OPTION_NULLABLE);
+  if (ret != GS_RESULT_OK)
+    return ret;
+  
+  ret = set_tableInfo(store, "pagg_tab_p1", &pagg_tab_p1,
+                  4,
+                  "t", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NOT_NULL,
+                  "a", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NOT_NULL,
+                  "b", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NULLABLE,
+                  "c", GS_TYPE_STRING,      GS_TYPE_OPTION_NULLABLE);
+  if (ret != GS_RESULT_OK)
+    return ret;
+  
+  ret = set_tableInfo(store, "pagg_tab_p2", &pagg_tab_p2,
+                  4,
+                  "t", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NOT_NULL,
+                  "a", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NOT_NULL,
+                  "b", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NULLABLE,
+                  "c", GS_TYPE_STRING,      GS_TYPE_OPTION_NULLABLE);
+  if (ret != GS_RESULT_OK)
+    return ret;
+  
+  ret = set_tableInfo(store, "pagg_tab_p3", &pagg_tab_p3,
+                  4,
+                  "t", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NOT_NULL,
+                  "a", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NOT_NULL,
+                  "b", GS_TYPE_INTEGER,     GS_TYPE_OPTION_NULLABLE,
+                  "c", GS_TYPE_STRING,      GS_TYPE_OPTION_NULLABLE);
+  if (ret != GS_RESULT_OK)
+    return ret;
+
+  /* For testing */
+  table_info simple;
+  set_tableInfo(store, "simple", &simple,
+                  2,
+                  "id", GS_TYPE_INTEGER, GS_TYPE_OPTION_NOT_NULL,
+                  "t", GS_TYPE_STRING, GS_TYPE_OPTION_NULLABLE);
+  if (ret != GS_RESULT_OK)
+    return ret;
+  
+  /* Release the resource */
+  gsCloseGridStore(&store, GS_TRUE);
+  
+  return GS_RESULT_OK;
+}
+
+/* Main funtion */
+int main(int argc, char *argv[])
```

Comment:
Please implement the argument checking.
If the argument is not correct, please output the usage.